clappr/src/main/kotlin/io/clappr/player/plugin/control/MediaControl.kt

```diff
@@ -74,7 +74,7 @@ open class MediaControl(core: Core, pluginName: String = name) :
 
     private val controlsPanel by lazy { view.findViewById(R.id.controls_panel) as RelativeLayout }
 
-    private val topCenterPanel by lazy { view.findViewById(R.id.top_center) as RelativeLayout }
+    private val topCenterPanel by lazy { view.findViewById(R.id.top_center) as LinearLayout }
```

Comment:
Using the more general interface `ViewGroup` should be enough in this context.

```suggestion
    private val topCenterPanel by lazy { view.findViewById<ViewGroup>(R.id.top_center) }
```

Reply:
Done 44a9d210